taxi/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.contrib.auth import get_user_model
+
+from taxi.models import Car, Manufacturer
+
+
+class ManufacturerForm(forms.ModelForm):
+    class Meta:
+        model = Manufacturer
+        fields = ("name", "country", )
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+        required=False
+    )
+
+    class Meta:
+        model = Car
+        fields = ("model", "manufacturer", "drivers", )
```

Comment:
You generics will implement this forms under the hood, so u don't need to create it by yourself

Reply:
Yes, I know that. I just added a widget for the list of drivers via a form, I thought it would be better than overriding get_form in several classes.